src/openms/source/CHEMISTRY/AASequence.cpp

```diff
@@ -1377,7 +1380,12 @@ namespace OpenMS
   void AASequence::parseString_(const String& pep, AASequence& aas,
```

Comment:
you could pass `pep` by value (since its copied internally anyways)

Reply:
I agree, that could be more efficient (not here though): https://stackoverflow.com/questions/270408/is-it-better-in-c-to-pass-by-value-or-pass-by-constant-reference